tests/_runtime/test_control_flow.py

```diff
@@ -12,9 +12,10 @@ def test_stop_false(k: Kernel) -> None:
     k.run(
         [
             ExecutionRequest(
-                "0", "import marimo as mo; x = 0; mo.stop(False); y = 1"
+                cell_id="0",
+                code="import marimo as mo; x = 0; mo.stop(False); y = 1",
             ),
-            ExecutionRequest("1", "z = y + 1"),
+            ExecutionRequest(cell_id="1", code="z = y + 1"),
```

Comment:
It seems like pydantic doesn't allow positional arguments, so I had to update this test and others.